docs/parsing_statistics.md

```diff
@@ -0,0 +1,188 @@
+# Parsing Statistics
+
+Shadow logs simulator heartbeat messages that contain useful system information for each virtual node in the experiment. For example, Shadow logs the number of bytes sent/received, number of bytes allocated/deallocated, CPU usage, etc. You can parse these heartbeat log messages to get insight into the simulation. Details of these heartbeat messages can be found [here](log_format.md#heartbeat-messages).
+ 
+## Generating Traffic
+
+This example uses the [TGen traffic generator](https://github.com/shadow/tgen) to generate network traffic between hosts, and then shows results from Shadow's heartbeat messages.
+
+TGen is capable of modeling generic behaviors with an action-dependency graph in the standard GraphML format. If you don't have it installed, you can follow the [instructions here](https://github.com/shadow/tgen/#setup). The following example runs TGen with 10 clients that each download 10 files from a set of 5 servers over a simple network topology.
```

Comment:
I only see 1 client and 1 server below? Should there be a `count` field?

Reply:
Yeah, I don't think the documentation follows the example. I changed the `count` filed in the TGen GraphML from 100 to 10, and added `count: 10` to the client host in the Shadow config yaml. I also changed the documentation to say 1 server.